Fancyauth.Model/FancyContextBase.cs

```diff
@@ -25,6 +26,13 @@ protected override void OnModelCreating(DbModelBuilder modelBuilder)
 
             modelBuilder.Entity<Song>()
                 .HasMany(x => x.AdditionalInterprets).WithMany().Map(x => x.ToTable("Songs_AdditionalInterprets"));
+
+            modelBuilder.Entity<PersistentGuest>()
+                .HasMany(x => x.Godfathers).WithMany(x => x.Godfatherships).Map(x => x.ToTable("PG_Godfathers"));
```

Comment:
Why no cascading delete of godfathers on deletion of Persistent Guest?  This could result in inconsistencies.